Testing/functional/tests/cypress/integration/e2e/user/vaccineCardQuickLink.js

```diff
@@ -0,0 +1,123 @@
+const { AuthMethod } = require("../../../support/constants");
+
+const homePath = "/home";
+
+const encounterModule = "Encounter";
+const immunizationModule = "Immunization";
+const laboratoryModule = "Laboratory";
+const allLaboratoryModule = "AllLaboratory";
+const medicationModule = "Medication";
+const medicationRequestModule = "MedicationRequest";
+const noteModule = "Note";
+const vaccinationStatusModule = "VaccinationStatus";
+
+const encounterTitle = "Health Visits";
+const immunizationTitle = "Immunizations";
+const laboratoryTitle = "COVID‑19 Tests";
+const vaccineCardTitle = "BC Vaccine Card";
+
+const addQuickLinkButtonSelector = "[data-testid=add-quick-link-button]";
+const addQuickLinkCheckboxSelector =
+    "[data-testid=quick-link-modal-text] input[type=checkbox]";
+const addQuickLinkCancelButtonSelector =
+    "[data-testid=cancel-add-quick-link-btn]";
+const addQuickLinkSubmitButtonSelector = "[data-testid=add-quick-link-btn]";
+const addQuickLinkModalTextSelector = "[data-testid=quick-link-modal-text]";
```

Comment:
Is this constant 'addQuickLinkModalTextSelector' being used? If not, please remove it.